deploy/operator.yaml

```diff
@@ -52,3 +52,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "managed-upgrade-operator"
+          livenessProbe:
+            tcpSocket:
+              port: 8686
+            initialDelaySeconds: 5
```

Comment:
Maybe let's increase this to 10 or something. 5 seconds seemed to be cutting it pretty close during the tests we did during the meet.

Reply:
Done.